Makefile

```diff
@@ -13,7 +13,7 @@ cover:
 .PHONY: lint
 lint:
 	@echo "==> 👕 Linting"
-	@poetry run black .
+	@poetry run ruff format draco jupyterlite pyodide docs
 	@poetry run ruff .
```

Comment:
Can we delete this line now?

Reply:
You mean `poetry run ruff .`? I think that's still necessary to do the actual linting.

Reply:
I see. Because formatting is only catching a subset of things. Sounds good.